src/main/resources/com/rultor/agents/daemons/end.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+#@todo #754:30min Remove the deprecation code from the shell.
+# Move the code to Java. May be
```

Comment:
@sebing why not just go by the xpath on the xml that is available in `com.rultor.agents.daemons.EndsDaemon`, no need for instantiating a `Profile`,
just work by analogy from the Profile logic and how it interprets the XML,  other examples of how this looks are all over the codebase :)